pkg/console/controllers/thanosquerierproxyserviceresolver/controller.go

```diff
@@ -0,0 +1,130 @@
+package thanosquerierproxyserviceresolver
```

Comment:
this pkg is the same as `pkg/console/controllers/advancedclustermanagement/thanosquerierproxyserviceresolver/controller.go`